Add appwrite+cordova app

arnu515 opened the pull request:
## What does this PR do?

Added Appwrite + Cordova demo. I've created a simple instagram clone called "instantgram" with Appwrite, Cordova and React (with vite).

## Test Plan

There is a tutorial in the README in the source code on how to set up the project

## Related PRs and Issues

appwrite/appwrite#4218

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

arnu515 commented:
Hey can I please get a review before the end of hacktoberfest?

PineappleIOnic commented:
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

stnguyen90 commented on README.md:
```diff
@@ -211,6 +212,9 @@ Share your apps here! Submit a pull request!
 * **Appwrite + Angular SimpleGrocery** - A simple app to create fast shopping-list. [Source Code](https://github.com/ffex/simple-grocery-demo)
 
+### Cordova
+* **Appwrite + Cordova + React** - A simple instagram clone built with Appwrite, Cordova and React (with vite). [Source Code](https://github.com/arnu515/instagram-clone-with-cordova-appwrite)
```
I tried running your project but got an error:

```
$ npx cordova run electron --nobuild
Current working directory is not a Cordova-based project.
```

Do you know what's wrong? Maybe I'm missing files like the files in the `www` folder?

arnu515 replied:
I believe you have to run `yarn build` first, so that the `www` folder gets created. I'll add it to the README

stnguyen90 reviewed (changes requested):
Great PR! 🤯 We left some comments during the review, please check them out.

stnguyen90 reviewed (changes requested):
Great PR! 🤯 We left some comments during the review, please check them out. 

1. Your screenshot says to create a bucket with ID `instantgram` but the [code](https://github.com/arnu515/instagram-clone-with-cordova-appwrite/blob/master/src/pages/new.tsx#L29) has `postmedia`
2. The [content placeholder](https://github.com/arnu515/instagram-clone-with-cordova-appwrite/blob/master/src/pages/new.tsx#L114) says I can use markdown, but it doesn't render as markdown 🧐

arnu515 commented:
> Your screenshot says to create a bucket with ID instantgram but the [code](https://github.com/arnu515/instagram-clone-with-cordova-appwrite/blob/master/src/pages/new.tsx#L29) has postmedia

I've fixed it now

> The [content placeholder](https://github.com/arnu515/instagram-clone-with-cordova-appwrite/blob/master/src/pages/new.tsx#L114) says I can use markdown, but it doesn't render as markdown monocle_face

I must have forgotten 😅. I've removed the notice now

stnguyen90 reviewed (approved):
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

christyjacob4 commented:
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)
